sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOptions.cs

```diff
@@ -16,5 +16,17 @@ public class AnalyzeHealthcareEntitiesOptions : TextAnalyticsRequestOptions
         public AnalyzeHealthcareEntitiesOptions()
         {
         }
+
+        /// <summary>
+        /// Optional display name for the operation.
+        /// </summary>
+        public string DisplayName { get; set; }
+
+        internal new AnalyzeHealthcareEntitiesOptions CheckSupported(TextAnalyticsClientOptions.ServiceVersion version)
```

Comment:
Why `new`?  Why not make this `private protected virtual` if we're going to use it like this?

Reply:
Because the return type changes. I considered making this generic, but not sure that offers any benefit over this. `call` is cheaper than `callvirt`, and with this approach you know that whatever type you call this method (convention by name) is what you get back. Thoughts?

Reply:
Why do we need to return it?

Reply:
To be able to call it inline, but honestly that's just a nice-to-have. I just have to go to block bodies now. :)